conf/messages.cy

```diff
@@ -199,7 +199,7 @@ cf.cash-account.detail.link.post = ar GOV.UK
 cf.cash-account.help-and-support.link.text.pre = Gallw ch
 cf.cash-account.help-and-support.link.text.pre.v2 = Gallwch
 cf.cash-account.help-and-support.link.text.post = os oes angen help arnoch
-cf.cash-account.help-and-support.link.text = gysylltu â chanolfan cyfrifyddu CThEF (yn agor tab newydd)
+cf.cash-account.help-and-support.link.text = gysylltu â chanolfan cyfrifyddu CThEF
```

Comment:
I'm assuming that the (yn agor tab newydd) is actually added twice as exists separately when the msg is built?